zipkin-storage/mysql-v1/src/test/java/zipkin2/storage/mysql/v1/ZipkinMySQLContainer.java

```diff
@@ -14,47 +14,24 @@
 package zipkin2.storage.mysql.v1;
 
 import com.github.dockerjava.api.command.InspectContainerResponse;
+import java.sql.Connection;
+import java.sql.SQLException;
 import org.mariadb.jdbc.MariaDbDataSource;
-import org.rnorth.ducttape.unreliables.Unreliables;
-import org.testcontainers.containers.ContainerLaunchException;
 import org.testcontainers.containers.GenericContainer;
-import org.testcontainers.jdbc.ext.ScriptUtils;
-import org.testcontainers.shaded.com.google.common.io.Resources;
+import org.testcontainers.jdbc.ContainerLessJdbcDelegate;
 
-import javax.script.ScriptException;
-import java.io.IOException;
-import java.net.URL;
-import java.nio.charset.Charset;
-import java.sql.Connection;
-import java.sql.SQLException;
-import java.util.concurrent.TimeUnit;
+import static org.testcontainers.ext.ScriptUtils.runInitScript;
 
-public class ZipkinMySQLContainer extends GenericContainer<ZipkinMySQLContainer> {
+final class ZipkinMySQLContainer extends GenericContainer<ZipkinMySQLContainer> {
 
   MariaDbDataSource dataSource;
 
-  public ZipkinMySQLContainer(String version) {
+  ZipkinMySQLContainer(String version) {
     super("openzipkin/zipkin-mysql:" + version);
-
     withExposedPorts(3306);
-
-    setWaitStrategy(new AbstractWaitStrategy() {
-      @Override
-      protected void waitUntilReady() {
```

Comment:
this basically never succeeded, which delayed container startup ignoring the fact that the init scripts never ran